Assets/Features/Workers/AssignableWorker.cs

```diff
@@ -123,13 +130,25 @@ public void MoveTo(Vector3Int target, Action onReachedPosition, Action onDone)
         /// </summary>
         protected bool CheckTaskExists(Vector3Int target, Action onDone)
         {
-            if (_grid.Get(target) == -1)
+            if (GameManager.Instance.GridManager.Grid.Get(target) == -1)
```

Comment:
-1 doesn't tell me anything about the state of the object. Could be beneficial to create a variable or something else that represents the -1 state, making it easier to read

Reply:
Oh and I don't know what you're trying to get with the `Get()` function... A state? An object? A cow?